src/main/resources/application.yml

```diff
@@ -110,10 +110,10 @@ delius-integration-context:
       nsi-mapping:
         nsi-status: INPROG
         service-category-to-nsi-type:
-          "[Accommodation]": CRS01
-          "[Education, Training and Employment (ETE)]": CRS02
-          "[Finance, Benefits and Debt (FBD)]": CRS03
-          "[Dependency and Recovery (D&R)]": CRS04
+          428ee70f-3001-4399-95a6-ad25eaaede16: CRS01
```

Comment:
I'm a bit confused why you need these GUIDs can't you just use the right hand code?

Reply:
We are translating from interventions service categories to Delius and it was thought that the description field was too fragile and the only handle remaining was the UUID.

Reply:
So those the calling service passthrough something like`428ee70f-3001-4399-95a6-ad25eaaede16` in the API request?